src/lib/application/files/ts/src/main.ts

```diff
@@ -1,8 +1,11 @@
 import { NestFactory } from '@nestjs/core';
+import { INestApplication, INestExpressApplication } from '@nestjs/common';
 import { AppModule } from './app.module';
 
-async function bootstrap() {
-  const app = await NestFactory.create(AppModule);
+async function bootstrap(): Promise<void> {
+  const app: INestApplication & INestExpressApplication = await NestFactory.create(AppModule);
```

Comment:
could you get rid of these additional typings? we'll change it very soon in the core

Reply:
Done.